e2e/env/env.go

```diff
@@ -14,12 +14,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	virtv1 "kubevirt.io/api/core/v1"
-	// Placement
-	ocmv1b1 "open-cluster-management.io/api/cluster/v1beta1"
-	// ManagedClusterSetBinding
-	ocmv1b2 "open-cluster-management.io/api/cluster/v1beta2"
-	// ClusterClaim
-	ocmv1a1 "open-cluster-management.io/api/cluster/v1alpha1"
+	ocmv1a1 "open-cluster-management.io/api/cluster/v1alpha1" // ClusterClaim
+	ocmv1b1 "open-cluster-management.io/api/cluster/v1beta1"  // Placement
+	ocmv1b2 "open-cluster-management.io/api/cluster/v1beta2"  // ManagedClusterSetBinding
```

Comment:
So if we want to revert this change, we need to also revert the doc change?

Please do ONE logic change per commit.